.github/ISSUE_TEMPLATE/spec_issue.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Specification Review Issue
+about: Start a request for a specification release
+title: ''
+label: Release Review
+assignees: ''
+---
+
+## Specification issue template
+When creating a specification project lifecycle review, create issues in the  with the content defined as follows.
+- [ ] Specification name and version
+- [ ] Add the label `Release Release`
```

Comment:
Need the other valid labels:  `Creation Review`, `Plan Review`, `Progress Review`, `Release Review`